CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## Changed
 - [#2652](https://github.com/plotly/dash/pull/2652) dcc.Clipboard supports htm_content and triggers a copy to clipboard when n_clicks are changed
 - [#2721](https://github.com/plotly/dash/pull/2721) Remove ansi2html, fixes [#2613](https://github.com/plotly/dash/issues/2713)
+- [#2734](https://github.com/plotly/dash/pull/2734) Configure CI for Python 3.10 [#1863](https://github.com/plotly/dash/issues/1863)
```

Comment:
FYI we just made a release, so if you rebase/merge we'll need to make a new `UNRELEASED` section